File: replace_ip.py
import os

old_ip1 = "165.227.89.199"
new_ip1 = "165.227.89.199"

old_ip2 = "165-227-89-199"
new_ip2 = "165-227-89-199"

folder = r"d:\noor tela\CloudRad"

updated_count = 0

for root, dirs, files in os.walk(folder):
    if ".git" in root or "node_modules" in root or "__pycache__" in root:
        continue
    for file in files:
        if file.endswith((".js", ".jsx", ".py", ".yml", ".yaml", ".conf", ".sh", ".ps1", ".md")):
            file_path = os.path.join(root, file)
            try:
                with open(file_path, "r", encoding="utf-8") as f:
                    content = f.read()
                
                if old_ip1 in content or old_ip2 in content:
                    content = content.replace(old_ip1, new_ip1)
                    content = content.replace(old_ip2, new_ip2)
                    
                    with open(file_path, "w", encoding="utf-8", newline='') as f:
                        f.write(content)
                    print(f"Updated: {file_path}")
                    updated_count += 1
            except Exception as e:
                pass

print(f"Done! Updated {updated_count} files.")

File: replace_ip.js
const fs = require('fs');
const path = require('path');

const oldIp1 = "165.227.89.199";
const newIp1 = "165.227.89.199";

const oldIp2 = "165-227-89-199";
const newIp2 = "165-227-89-199";

function walkDir(dir) {
    const files = fs.readdirSync(dir);
    for (const file of files) {
        const fullPath = path.join(dir, file);
        if (fullPath.includes('.git') || fullPath.includes('node_modules')) continue;
        
        const stat = fs.statSync(fullPath);
        if (stat.isDirectory()) {
            walkDir(fullPath);
        } else {
            if (fullPath.match(/\.(js|jsx|py|yml|yaml|conf|sh|ps1|md)$/)) {
                let content = fs.readFileSync(fullPath, 'utf8');
                if (content.includes(oldIp1) || content.includes(oldIp2)) {
                    content = content.split(oldIp1).join(newIp1);
                    content = content.split(oldIp2).join(newIp2);
                    fs.writeFileSync(fullPath, content, 'utf8');
                    console.log("Updated: " + fullPath);
                }
            }
        }
    }
}

walkDir("d:\\noor tela\\CloudRad");
